### PR TITLE
switch to new registrator and drop CI support for julia 0.7

### PR DESCRIPTION
**Pull request by johnnychen94**
more details in https://github.com/JuliaImages/Images.jl/issues/798

also falls back to default travis script to test

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
+ImageTransformations = "02fcd773-0e25-5acc-982a-7f6622650795"
+
+[compat]
+ColorTypes = ">= 0.3.2"
+Crayons = ">= 0.1.0"
+ImageCore = ">= 0.7"
+ImageTransformations = ">= 0.3.0"
+julia = ">= 0.7"
+
```
Comment (Evizero) [inline Project.toml:17]: I think we'll need this to make the test-time dependencies work

```suggestion
[targets]
test = ["Test", "FixedPointNumbers", "TestImages", "OffsetArrays", "CoordinateTransformations", "Rotations", "ImageMagick"]
```

File: Project.toml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+[extras]
+CoordinateTransformations = "150eb455-5306-5404-9cee-2592286d6298"
+FixedPointNumbers = "53c48c17-4a7d-5ca2-90c5-79b7896eea93"
+ImageMagick = "6218d12a-5da1-5696-b52f-db25d2ecc6d1"
+OffsetArrays = "6fe1bfb0-de20-5000-8ca7-80f57d26f881"
+QuartzImageIO = "dca85d43-d64c-5e67-8c65-017450d5d020"
```
Comment (johnnychen94) [inline Project.toml]: I guess I made a mistake here including `QuartzImageIO`

**Comment by johnnychen94** [comment]
ping @Evizero

**Comment by johnnychen94** [comment]
don't forget the version bump 😄 

`@JuliaRegistrator register()`

**Pull request merged**